02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,173 @@
+              <li class="guitar"><span id="s2">Nature Walks</span></li>
+              <div id="hobbiessidelist">
+                <li class="guitar"><span id="s1">Modern / Obscure Science</span></li>
+                  <ul>
+                    <li class="guitar"><span id="s4">~Quantum Physics is crazy, yo!</span></li>
+                    <li class="guitar"><span id="s5">~Prehistory is also cool</span></li>
+                  </ul>
+              </div>
+            </ul>
+          </div>
+        </div>
+        <br><br>
+        <!--Create rollover image effect with 3 diff shows/books/movies-->
+        <div class="favshows">
+          <p>Favorite TV Shows</p>
+        </div>
+        <div id="tvtagline">
+          <p>~You may notice a trend~</p>
+        </div>
+        <div class="showpics">
+          <img src="http://thegeekiary.com/wp-content/uploads/2015/07/rick-and-morty-season-2.jpg" alt="Image of Rick and Morty" title="Rick & Morty" style="height:300px"/>
+          <img src="http://vignette2.wikia.nocookie.net/simpsons/images/c/c0/Futurama.jpg/revision/latest?cb=20120111030407" alt="Image of Futurama" title="Futurama" style="height:300px"/>
+          <img src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcQwS6Z3lE0OGfqjctWtl_Wa1KnB9nEV-npK_H8cx7jHLQk4DFqbPw" alt="Image of American Dad" title="American Dad" style="height:300px"/>
+          <img src="http://s.sidereel.com/tv_shows/42681/giant_2x/family-guy.jpg" alt="Image of Family Guy" title="Family Guy" style="height:300px"/>
+          <img src="http://www.gunaxin.com/wp-content/uploads/2011/05/simpsons.jpg" alt="Image of The Simpsons" title="The Simpsons" style="height:300px"/>
+          <img src="https://assets.fxnetworks.com/cms/prod/shows/archer/photos/archer/web/web_largecoverart_series_archer_s6_270x398.jpg" alt="Image of Archer" title="Archer" style="height:300px"/>
+          <img src="http://studentsforliberty.org/wp-content/uploads/2014/09/South_Park_banner.png" alt="Image of South Park" title="South Park" style="height:300px"/>
+        </div>
+        <div class="future">
+          <p id="futurehead">My Goals</p>
+        </div>
+        <ul class="goals">
+          <li id="s1">I want to live near mountains</li>
+          <li id="s2">I want to be my own boss</li>
+          <li id="s3">I want to spend time abroad</li>
+          <li id="s4">I want 5 dogs (and no kids)</li>
+          <li id="s5">I want to go to space</li>
+          <li id="s6">I want to learn how to cook</li>
+          <li id="s1">I want to write a book</li>
+          <li id="s2">I want to learn to play the Saxophone</li>
+          <li id="s3">I want to be a Programming Guru</li>
+          <li id="s4">I want to improve the world</li>
+        </ul>
+
+
+<!--Put in blocks saying Austin, GT, etc
+    then give a hover effect to show another
+    image -->
+
+<!--Make sure all images have alt and title attributes-->
+
+<!--Download font pack to make more fancy-->
 
 </body>
```

Comment:
Outstanding work here. You spent a lot of time and put a lot of thought into this and it shows. Keep pushing yourself and you will only get better. I'd love to see what a Ninja-Cowboy costume looks like.